src/EditorFeatures/CSharpTest/InlineDeclaration/CSharpInlineDeclarationTests_FixAllTests.cs

```diff
@@ -94,6 +95,212 @@ void M()
     {
         GetExeAndArguments(useCmdShell, executable, arguments, out string finalExecutable, out string finalArguments);
     }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineDeclaration)]
+        [WorkItem(28323, "https://github.com/dotnet/roslyn/issues/28323")]
+        public async Task FixAllInDocument4()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        int {|FixAllInDocument:i1|}; int i2;
+        int.TryParse(v, out i1);
+        int.TryParse(v, out i2);
+    }
+}",
+@"class C
+{
+    void M()
+    {
+        int.TryParse(v, out int i1);
+        int.TryParse(v, out int i2);
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineDeclaration)]
+        [WorkItem(28323, "https://github.com/dotnet/roslyn/issues/28323")]
+        public async Task FixAllInDocument5()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        int dummy; int {|FixAllInDocument:i1|}; int i2;
+        int.TryParse(v, out i1);
+        int.TryParse(v, out i2);
+        dummy = 42;
+    }
+}",
+@"class C
+{
+    void M()
+    {
+        int dummy;  
+        int.TryParse(v, out int i1);
+        int.TryParse(v, out int i2);
+        dummy = 42;
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineDeclaration)]
+        [WorkItem(28323, "https://github.com/dotnet/roslyn/issues/28323")]
+        public async Task FixAllInDocument6()
+        {
+            await TestInRegularAndScriptAsync(
+@"class C
+{
+    void M()
+    {
+        int {|FixAllInDocument:i1|}; int dummy; int i2;
```

Comment:
can we have some tests with comments? like /* 0 */int /* 1 */ dummy /* 2 */; /* 3*/